### PR TITLE
small improvements

### PR DESCRIPTION
**Pull request by rxbn**
- use switch statements where applicable
- validate `cloudflare.defaults.type`

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,13 @@
 # Changelog
 
+## [v1.1.1](https://github.com/containeroo/SyncFlaer/tree/v1.1.1) (2021-02-XX)
```
Comment (rxbn) [inline CHANGELOG.md:3]: update date

File: internal/additionalRecords.go [hunk 2]
```diff
@@ -20,11 +20,14 @@ func GetAdditionalRecords(userRecords []cloudflare.DNSRecord) []cloudflare.DNSRe
...
-			if additionalRecord.Type == "A" {
+			switch additionalRecord.Type {
+			case "A":
 				additionalRecord.Content = currentIP
-			}
-			if additionalRecord.Type == "CNAME" {
+			case "CNAME":
 				additionalRecord.Content = config.Cloudflare.ZoneName
+			default:
+				log.Errorf("%s is an unsupported type, only A or CNAME are supported", additionalRecord.Content)
```
Comment (rxbn) [inline internal/additionalRecords.go:29]:
```suggestion
				log.Errorf("%s is an unsupported type, only A or CNAME are supported", additionalRecord.Type)
```

**Review by rxbn — COMMENTED** [review]
LGTM

**Review by rxbn — COMMENTED** [review]
LGTM

**Comment by gi8lino** [comment]
LGTM
maybe log if defaults are set/used

**Review by rxbn — COMMENTED** [review]
LGTM

**Comment by gi8lino** [comment]
LGTM

**Pull request merged**